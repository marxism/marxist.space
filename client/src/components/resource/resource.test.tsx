/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import Resource from '.'
import ResourceContent from './content'

it('renders without crashing', () => {
  // @ts-ignore
  shallow(<Resource />)
})

it('renders information about the resource', () => {
  // @ts-ignore
  const wrapper = shallow(<Resource />)
  expect(wrapper.contains(<ResourceContent />)).toEqual(true)
})
