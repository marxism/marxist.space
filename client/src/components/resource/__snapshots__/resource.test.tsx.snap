// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resource renders with all props 1`] = `
<div
  className="lazyload-placeholder"
  style={
    Object {
      "height": 51,
    }
  }
/>
`;

exports[`resource renders with minimum props 1`] = `
<div
  className="lazyload-placeholder"
  style={
    Object {
      "height": 51,
    }
  }
/>
`;
